test/pkg/integration/common/git-client.go

```diff
@@ -108,7 +123,7 @@ func (g GitClient) Commit(dir string, message string, all bool) error {
 		return err
 	}
 	if resp.ExitCode != 0 {
-		return xerrors.Errorf("commit returned rc: %d", resp.ExitCode)
+		return fmt.Errorf("commit returned rc: %d err: %v", resp.ExitCode, resp.Stderr)
```

Comment:
why fmt instead of xerrors? I believe xerrors is preferred?

Reply:
I check recently PR deprecated xerrors.